pretty_midi/containers.py

```diff
@@ -23,6 +23,9 @@ class Note(object):
     """
 
     def __init__(self, velocity, pitch, start, end):
+        if end < start:
+            raise ValueError("end paramater must be greater than start")
```

Comment:
Suggest "Note end time must be greater than start time" as a bit more informative